internal/version/check_update.go

```diff
@@ -6,18 +6,37 @@ package version
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
 
 	"github.com/Masterminds/semver/v3"
 
+	"github.com/elastic/elastic-package/internal/configuration/locations"
+	"github.com/elastic/elastic-package/internal/environment"
 	"github.com/elastic/elastic-package/internal/github"
 	"github.com/elastic/elastic-package/internal/logger"
 )
 
 const (
 	repositoryOwner = "elastic"
 	repositoryName  = "elastic-package"
+
+	latestVersionFile    = "latestVersion"
+	defaultCacheDuration = "30m"
```

Comment:
Define it directly as duration?
```suggestion
	defaultCacheDuration = 30 * time.Minute
```